aws-network-spe-py/__main__.py

```diff
@@ -48,6 +62,7 @@
         "faucet_pubkey": faucet_key.public_key,
         "bootstrap_validator_stake_lamports": 10000000000,  # 10 SOL
         "enable_warmup_epochs": True,
+        "slots_per_epoch": 8192,
```

Comment:
Dropping this change when this is merged.

https://github.com/abklabs/svmkit-examples/pull/68